src/l10n.json

```diff
@@ -247,8 +247,9 @@
     "thumbnail.by": "by",
     "report.error": "Something went wrong when trying to send your message. Please try again.",
     "report.project": "Report Project",
-    "report.projectInstructions": "From the dropdown below, please select the reason why you feel this project is disrespectful or inappropriate or otherwise breaks the {CommunityGuidelinesLink}.",
+    "report.projectInstructions": "When you send a report, it lets the Scratch Team know about projects that break the {CommunityGuidelinesLink}. Does something in this project break the {CommunityGuidelinesLink}? If you think it does, please tell us more.",
     "report.CommunityGuidelinesLinkText": "Scratch Community Guidelines",
+    "report.shortCommunityGuidelinesLinkText": "Community Guidelines",
```

Comment:
This is causing test to fail; use `general.guidelines`

Reply:
thanks for noticing that!